### PR TITLE
add a missing apostrophe

### PR DESCRIPTION
**Pull request by rebeccaskinner**

### CODE REVIEW

**Review by TikhonJelvis — APPROVED** [review]
Yay

**Review by romefeller — COMMENTED** [review]
Ok

**Review by romefeller — COMMENTED** [review]
Ok.

File: testimonials/mercury.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ companyName: Mercury
 logoURL: logos/mercury.png
 shortTestimonial: |
   Mercury is a bank for businesses. We use Haskell to meet our
-  customers high standards for correctness and security.
+  customers' high standards for correctness and security.
```
Comment (uzair-javed-1) [inline testimonials/mercury.yaml:6]: Customer is already a plural / Noun , so you can not make it to customers or customer's . so change it to customer .

**Review by uzair-javed-1 — COMMENTED** [review]
Customer is already a plural / Noun , so you can not make it to customers or customer's . so change it to customer .

File: testimonials/mercury.yaml (hunk 1 above)
Comment (uzair-javed-1) [reply testimonials/mercury.yaml:6]: ok i am gonna change it's to customer's

**Review by uzair-javed-1 — COMMENTED** [review]
ok i am adding a missing apostrophe

**Pull request merged**